### PR TITLE
[WIP!] New "LOOT for Mod Authors" section in the documentation

### PR DESCRIPTION
**Pull request by Freso**
To make it easier for mod authors to work with LOOT (and have LOOT work with their mod(s)), I've been meaning to write some documentation specifically for this segment, to make it as clear as possible what they can do to make LOOT get along with their mod(s) as best as possible.

It's in a *very* early stage, but suggestions and feedback are very welcome! The rendered page can be viewed at https://loot.readthedocs.io/en/mod-author-docs-section/app/modauthors.html

### CODE REVIEW

File: docs/app/modauthors.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+For plugin files, having a line with just something like "Version 2.1.3" in the plugin description [#snam_records]_ makes it the most obvious for LOOT's parsing, but LOOT tries to be smart and thus also recognises a number of other formats (e.g., "v2.1.3" or simply just "2.1.3") [#version_format]_. Since LOOT has no way of know what versioning scheme is being used, it will general compare versions using the `Semantic Versioning scheme`_, so it would also make it easier on LOOT if you follow that (or a scheme compatible with it).
+
+.. _`Semantic Versioning scheme`: https://semver.org/
+
+(Is there any other metadata that can be provided directly by the mod?)
```
Comment (WrinklyNinja) [inline docs/app/modauthors.rst]: Bash Tags can be given in the description field, though that's not really for LOOT. I can't think of anything else.

**Comment by Ortham** [comment]
@Freso what's the status of this?

**Comment by Ortham** [comment]
I don't remember closing this... I think it was because I deleted the `dev` branch.

**Comment by sibir-ine** [comment]
I think an introduction to LOOT & an FAQ specifically for mod authors is a really good idea, so I'd like to pick this up at some point. @Ortham, would you still be interested in something like this? If so, @Freso, would you mind if I picked this up from where you left it, or would you rather I started from scratch?

**Comment by Ortham** [comment]
I forgot to reply to this... I think it's still a good idea.

**Pull request closed without merging**